src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,15 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
+        Truck truck = new Truck();
```

Comment:
Look at the checklist. Let's use abstract reference instead of specific one 
![image](https://user-images.githubusercontent.com/68428976/129674573-64e10ec6-27aa-415a-a987-8c50817ba4dc.png)